modules/solid_mechanics/src/actions/DomainIntegralAction.C

```diff
@@ -402,14 +398,6 @@ DomainIntegralAction::act()
           av_name_stream << av_base_name << "_" << _ring_vec[ring_index];
           std::ostringstream pp_name_stream;
           pp_name_stream << pp_base_name << "_" << _ring_vec[ring_index];
```

Comment:
we're not using string streams for this purpose anymor (we have `MooseUtils::stringify()` now)